README.md

```diff
@@ -41,6 +41,11 @@ You can open an issue to request more crates.
 
 ## Run locally
 
+Install [wasm-pack](https://rustwasm.github.io/wasm-pack/installer/):
+```sh
+curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
+```
```

Comment:
We're currently using `wasm-pack 0.10.3` because of https://github.com/RReverser/wasm-bindgen-rayon/issues/9

```suggestion
Install [`wasm-pack`](https://rustwasm.github.io/wasm-pack):

```sh
cargo install wasm-pack@0.10.3 --locked
```
```